packages/b/buttercms.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "ButterCMS",
+  "description": "ButterCMS JS API Client",
+  "filename": "buttercms.min.js",
```

Comment:
As the CI error notes, the filename you've set here does not exist in the package files. I suspect you want `butter.js`?

Reply:
:grimacing: I have updated, sorry about that.